code/modules/guardian/abilities/major/explosive.dm

```diff
@@ -0,0 +1,87 @@
+#define UNREGISTER_BOMB_SIGNALS(A) \
+	do { \
+		UnregisterSignal(A, boom_signals); \
+		UnregisterSignal(A, COMSIG_PARENT_EXAMINE); \
+	} while (0)
+
+/datum/guardian_ability/major/explosive
+	name = "Explosive"
+	desc = "The guardian can, with a single touch, turn any inanimate object into a bomb."
+	cost = 4
```

Comment:
should probably lower this to 3

Reply:
this literally, if picked up, will either gib you, or blow off two of your limbs and kill you. it can also be remotely detonated, which can blow open doors and walls.

Reply:
it doesn't gib you. it does delimb you, though. good point.